### ISSUE TITLE
Lossless queues 3 and 4 are printed as lossy queues in syslog

### ISSUE THREAD

**Issue by nazeerhussainf** [Azure/sonic-buildimage#5910]
**Description**

**Steps to reproduce the issue:**
1. Topology :
Ixia1 ---- (Ethernet0) DUT (Ethernet4) ---- Ixia2
2. Create the topology as mentioned above.
3. Send traffic from Ixia1 on priority 0 to 7 to Ixia2.
4. Enable PFCWD with 200ms as detection and restoration time.
5. Send pause storm on priority 3 or priority 4 from Ixia2.
6. Observe the syslog for the following message.

Aug 20 13:08:43.583610 sonic WARNING swss#orchagent: :- pfcFrameCounterCheck: Got PFC 2232283451 frame(s) on lossy queue 3 port Ethernet4
[config_db.txt](https://github.com/Azure/sonic-buildimage/files/5530062/config_db.txt)
[syslog.txt](https://github.com/Azure/sonic-buildimage/files/5530063/syslog.txt)
Aug 20 13:08:43.583704 sonic WARNING swss#orchagent: :- pfcFrameCounterCheck: Got PFC 2232283450 frame(s) on lossy queue 4 port Ethernet4

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

**Comment by lguohan** [comment]
is this a typo?

**Comment by neethajohn** [comment]
No. That msg should only get printed if PFC frames are received on lossy queues. Need to understand why it is showing up for 3 and 4

**Comment by neethajohn** [comment]
This message gets printed for unset bits of the pfcMask associated with a port. Since 3 and 4 are lossless queues, bits 3 and 4 are set by default in the port pfcMask. During a pfc storm on a particular queue, we unset that queue from the pfcMask of the corresponding port which leads to this message getting printed for the queue that is currently stormed.
